io/net/iperf_test.py

```diff
@@ -194,6 +207,17 @@ def tearDown(self):
         """
         Killing Iperf process in peer machine
         """
+        cmd1 = "systemctl start firewalld.service"
+        cmd2 = "systemctl status firewalld.service"
```

Comment:
duplicate commands in setup and cleanup

Reply:
not using the duplicate cmds, modified as suggested.